LOGGING/src/ILogger/ApplicationInsightsLogger.cs

```diff
@@ -174,7 +174,14 @@ private void PopulateTelemetry<TState>(ISupportProperties telemetryItem, TState
                 {
                     foreach (KeyValuePair<string, object> item in stateDictionary)
                     {
-                        dict[item.Key] = Convert.ToString(item.Value, CultureInfo.InvariantCulture);
+                        if (item.Key == "{OriginalFormat}")
```

Comment:
Please add this to Changelog as some folks may have already taken a dependency on { being in the customproperty name.